lib/pf/web/dispatcher.pm

```diff
@@ -43,6 +43,9 @@ use pf::Portal::Session;
 use pf::web::externalportal;
 use pf::inline;
 
+use pfconfig::cached_scalar;
+tie our $portal_request_timeout, 'pfconfig::cached_scalar', 'resource::portal_request_timeout';
```

Comment:
This is so captiveportal in Catalyst doesn't have to use pf::config so its lighter